### PR TITLE
Removed ChatBot Functionality

### PR DESCRIPTION
**Pull request by Sarthak5598**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,6 +16,7 @@ slackeventsapi = "^3.0.1"
 slack-sdk = "^3.27.2"
 openai = "^1.35.7"
 cachetools = "^5.3.3"
+pre-commit = "^3.7.1"
```
Comment (arkid15r) [inline pyproject.toml:19]: Should go under dev section requirements.

**Review by arkid15r — COMMENTED** [review]
It's not recommended to have commented code in the code base. Git allows to use commit entities if you need to revert something.

File: pyproject.toml (hunk 1 above)
Comment (Sarthak5598) [reply pyproject.toml:19]: okay!

**Comment by Sarthak5598** [comment]
I commented it out as this functionality will be used in future, Is it still better to revert it ?

**Comment by arkid15r** [comment]
> I commented it out as this functionality will be used in future, Is it still better to revert it ?

I prefer to keep code clean and have comments for code explanation only. You can always dig git history and get those changes back when needed. It also may change and that functionality will be no longer needed but the commented code won't be taken care of.

**Comment by Sarthak5598** [comment]
okay will revert it , thank you!

**Pull request closed without merging**